src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_ReadOnlyStruct.cs

```diff
@@ -0,0 +1,675 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Symbols.Metadata.PE;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.CSharp.UnitTests;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using System.Collections.Immutable;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
```

Comment:
Example test: ReadOnlyStructSimpleLangVer in syntax tests

---
In reply to: [119929636](https://github.com/dotnet/roslyn/pull/19888#discussion_r119929636) [](ancestors = 119929636,119923247)